src/SFA.DAS.DATA.MANAGEMENT/Tables/AsDataPL_Comt_TransferRequest.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [AsData_PL].[Comt_TransferRequest]
+(
+	[Id]										[bigint]			NOT NULL,
+	[CommitmentId]								[bigint]			NOT NULL,
+	[TrainingCourses]							[nvarchar](max)		NOT NULL,
+	[Cost]										[money]				NOT NULL,
+	[Status]									[tinyint]			NOT NULL,
+	[TransferApprovalActionedByEmployerEmail]	[nvarchar](255)		NULL,
```

Comment:
Is this column needed Sarma TransferApprovalActionedByEmployerEmail - Looks PII to me, If not used better to exclude.

Reply:
Sarma confirmed that it's needed for Reporting